### PR TITLE
Update Grpc.DotNet.sln to use file scoped namespaces

### PR DESCRIPTION
**Pull request by JamesNK**
Automated by a fixer. Some minor cleanup in a test file that had multiple namespace scopes.

I will update sample solutions later.

Turn off whitespace when looking at changes. There isn't much to see.

### CODE REVIEW

File: src/Shared/NullableAttributes.cs [hunk 1]
```diff
@@ -16,109 +16,184 @@
 
 #endregion
 
+
+/* Unmerged change from project 'Grpc.AspNetCore.Server (netcoreapp3.0)'
```
Comment (BrennanConroy) [inline src/Shared/NullableAttributes.cs:20]: Fix?

**Comment by JamesNK** [comment]
@jtattermusch FYI. Existing PRs will have merge conflicts.

Any objections?

**Review by jtattermusch — APPROVED** [review]
LGTM assuming the only changes are whitespace changes and the use of scoped namespaces.

**Comment by JamesNK** [comment]
Sure. Shouldn't be difficult: https://github.com/grpc/grpc-dotnet/issues/1900

**Pull request merged**